04-FirstLab/README.md

```diff
@@ -1,28 +1,28 @@
 # First lab
 
 * Perform tasks below.
-* Watch the _PowerShell_ command line how it changes.
+* Watch how the command line is changing.
 * Use ```git status``` to see what is going on.
 
 # Task
 
-* Create in your repository following folders and files.
-* To files put some content per your preference:
+* Create the following folders and files in your repository.
+* Write some content per your preference into the files:
```

Comment:
I also rephrased this in PR #28.
What you have is also better than what was there before.  Either one.